SingularityService/src/main/java/com/hubspot/singularity/resources/HistoryResource.java

```diff
@@ -164,14 +164,15 @@ public SingularityDeployHistory getDeploy(
       @Parameter(required = true, description = "Request ID for deploy") @PathParam("requestId") String requestId,
       @Parameter(required = true, description = "Deploy ID") @PathParam("deployId") String deployId,
       @Parameter(description = "Maximum number of items to return") @QueryParam("count") Integer count,
-      @Parameter(description = "Which page of items to view") @QueryParam("page") Integer page) {
+      @Parameter(description = "Which page of items to view") @QueryParam("page") Integer page,
+      @Parameter(description = "Skip checking zookeeper, items that have not been persisted yet may not appear") @QueryParam("skipZk") Boolean skipZk) {
```

Comment:
Maybe add `@DefaultValue("true")`? Then you can switch the type to a primitive `boolean` and remove the ternary below

Reply:
Good call, I always forget that annotation exists